Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs

```diff
@@ -17,7 +17,7 @@ namespace ScaleMonk.Ads
     public class AdsProvidersHelper
     {
         const string iosAdsVersion = "1.3.0";
-        const string androidAdsVersion = "3.1.0";
+        const string androidAdsVersion = "4.0.0-alpha.1";
```

Comment:
I guess we'll change this when we do the public release, right?

Reply:
Yes, that's for sure.